docs/source/ci-cd.md

```diff
@@ -126,85 +118,41 @@ Each `build-scripts` folder includes `Readme.md`. Please check them for more det
+2. [**GHA `tag.yaml`**]
+    - Bumps version
+    - creates PR with the updated Version
+2. [**GHA `releasepr.yaml`**]
+    - Runs on the update Version PR
+    - Tests the code (e.g. DCO, CI testing, static code validation etc.).
+    - Builds packages.
+
+#### 2. Release Candidate and Release Acceptance
 
 **Note** If any of the following steps fails new release candidate should be prepared.
 
 1. [**Maintainer**]
-    - **Start release candidate pipeline manually**.
-2. [**build server**]
-    - Checkout the repository.
-    - Publish to PyPI as `X.Y.Z.rcN`.
-    - Bump version locally to `X.Y.Z`, commit and push as the `release commit` to remote.
-    - Build debian packages:
-        - for the project: source code version would be `X.Y.Z`, debian package version `X.Y.Z~rcN`;
-        - for the 3rd party dependencies missed in the official debian repositories.
-    - Publish the packages to `rc-latest` debian channel.
-    - [`indy-node`] Copy the package along with its dependencies (including `indy-plenum`)
-      from `rc-latest` to `rc` channel.
-    - [`indy-node`] Run system tests for the `rc` channel.
-    - Create **release PR** from `release-X.Y.Z` (that points to `release commit`) branch to `stable`.
-    - Notify maintainers.
-    - Wait for an approval to proceed. **It shouldn't be provided until `release PR` passes all necessary checks** (e.g. DCO, CI testing, maintainers reviews etc.).
-3. [**build server**]
-    - Run CI for the PR and notify GitHub.
-4. [**QA**]
-    - (_optional_) Perform additional testing.
-5. [**Maintainer**]
-    - Review the PR but **do not merge it**.
-    - If approved: let build server to proceed.
-    - Otherwise: stop the pipeline.
-6. [**build server**]
-    - If approved:
-        - perform fast-forward merge;
-        - create and push tag `vX.Y.Z`;
-        - Notify maintainers.
-    - Otherwise rollback `release commit` by moving `release-X.Y.Z` to its parent.
-
-#### 3. Publishing
-
-1. [**build server**] triggered once the `release PR` is merged
-    - Publish to PyPI as `X.Y.Z`.
-    - Download and re-pack debian package `X.Y.Z~rcN` (from `rc-latest` channel) to `X.Y.Z` changing only the package name.
-    - Publish the package to `rc-latest` debian channel.
-    - Copy the package along with its dependencies from `rc-latest` to `stable-latest` channel.
-    - [`indy-node`] Copy the package along with its dependencies (including `indy-plenum`) from `stable-latest` to `stable` channel.
-    - [`indy-node`] Run system tests for the `stable` channel.
-    - Notify maintainers.
+    - Wait till the PR with the updated Version number from the RC preperation step is created and the Pipeline has run its tests (`releasepr.yaml` run successfully).
+2. [**Maintainer**]
+    - If all checks passed and RC is approved merge the update version PR. It will kick of the `Releasepr.yaml`Pipeline, which creates a Github Release and publishes the packages.
```

Comment:
```suggestion
    - If all checks passed and RC is approved merge the update version PR. It will kick of the `publishRelease.yaml` Pipeline, which creates a Github Release and publishes the packages.
```